pandapower/plotting/plotly/simple_plotly.py

```diff
@@ -212,27 +216,38 @@ def _simple_plotly_generic(net, respect_separators, use_branch_geodata, on_map,
                                       respect_separators,
                                       color=branch_color, width=branch_width,
                                       infofunc=hoverinfo)
-    trans_trace = []
+    trans_trace1 = []
+    trans_trace2 = []
+    ext_grid_trace = []
     # ----- Trafos ------
     if 'trafo' in net:
         hoverinfo = hoverinfo_func(net, element=trans_element)
-        trans_trace += create_trafo_trace(net, color=trafo_color, width=branch_width * 5,
+        trans_trace1 = create_trafo_trace(net, color=trafo_color, width=branch_width * 5,
```

Comment:
can you please name them trans_trace and trans_trace3w?

Reply:
Good point. I've changed it accordingly.